test/patch/element.spec.js

```diff
@@ -113,6 +113,17 @@ describe('element', function () {
     expect(eventListener.handleEvent).not.toHaveBeenCalled();
   });
 
+  it('should have no effect while calling addEventListener / removeEventListener without listener', function () {
+    var eventListener;
+    expect(function() {
```

Comment:
Could you please
- removed eventListener - only used undefined / null
- add a test for ie {a: 1} - you had a question about that
- split add & remove in 2 different tests